yarn.lock

```diff
@@ -1341,17 +1341,6 @@
     react-mixin "^3.0.5"
     react-timer-mixin "^0.13.3"
 
-"@expo/react-native-navigator@0.5.2":
```

Comment:
Something weird is going on here. I deleted this in a prior commit, check your Git state.